manifests/rally-core-addon.yml

```diff
@@ -7,3 +7,4 @@ artifacts:
   - web-ext-artifacts/rally_core.xpi
 addon-type: privileged
 install-type: npm
+docker-image: node-15
```

Comment:
Let's keep this a staging-only change to test; let's not bump rally-core unless they actually want to move to node 15.